[Style Guide] Content Strategy (#9846)
cloudflare/cloudflare-docs: content/style-guide/content-strategy/documentation-content-strategy/content-types/faq.md

---
pcx_content_type: concept
title: FAQ

---

# FAQ

## Purpose

The purpose of an FAQ is to provide simple answers to common questions.

## Tone

Guiding, straightforward, educational, authoritative

## content_type

faq

## Overview

> A Frequently Asked Question (FAQ) page is a priority area for SEO and digital marketing, and can be a simple way to improve navigation for users.

Reviewer: ```suggestion
A Frequently Asked Questions (FAQ) page is a priority area for SEO and digital marketing, and can be a simple way to improve navigation for users.
```

#### Main FAQ page

[**Title**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/titles/): FAQ

[**Context**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/titles/) (page): an introductory paragraph on the section and what users can expect from it.

Section titles

[**Context**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/titles/) (section): a one-liner describing what users will find in that sub-section

Button: a button leading to the subpage with the actual questions

#### Child FAQ page

Breadcrumbs back to the main FAQ page

[**Title**](/style-guide/content-strategy/documentation-content-strategy/component-attributes/titles/): corresponds to the section header from the Main FAQ page

Questions, answers

## Question types

+ **Yes/No**
    + Can I do something?
    + Can the product do something?
+ **Procedural**
    + How do I do something?
    + How does something work?
    + How is something measured/counted?
+ **Definitions**

>     + What (is|are) ...?

Reviewer: ```suggestion
    + What (is/are) ...?
```

+ **Scenarios**
    + What if ...?
+ **Troubleshooting**
    + I see `<ERROR>`.
    + `<PRODUCT>` fails, shows errors,

## Guidelines

### General

Write the questions from the customer's POV, so use the first person.

> ✅Can I use wildcards when creating policies?
> 
> ❌Can users use wildcards when creating policies?
> 

Reviewer: ```suggestion
✅ Can I use wildcards when creating policies?

❌ Can users use wildcards when creating policies?

```

### Yes/No

With this question type, users want to inquire about capabilities. Does the product enable them to do something? Can the product do something?

+ **Question**
    + Yes/no questions should start with structures like:
        + Can I....

>         + (Can|Does) the product...
> + **Answer**
>     + Start the answer with Yes/No.
>         + ✅Yes. Cloudflare Access supports several providers simultaneously.
>         + ❌Cloudflare Access supports several providers simultaneously.

Reviewer: ```suggestion
        + (Can/Does) the product...
+ **Answer**
    + Start the answer with Yes/No.
        + ✅ Yes. Cloudflare Access supports several providers simultaneously.
        + ❌ Cloudflare Access supports several providers simultaneously.
```

    + Always follow with a short contextualization. Give the user all the information they need.

### Procedural

This question type addresses doubts regarding how to achieve a goal with the product, or how the product works. They should normally be addressed by either tutorials or how-tos in the main docs, but it's worth calling out some commonly asked procedural questions in the FAQ too, and linking back to other areas of the docs.

+ Question
    + Procedural questions should start with structures like:
        + How do I....
        + How does the product...
        + How does ... work?
+ Answer
    + Give concise but complete answers

>     + **Link out to relevant docs** (tutorials, how-tos, even blog posts) for more in-depth info

Reviewer: ```suggestion
    + **Link out to relevant docs** (tutorials, how-tos, even blog posts) for more in-depth information
```

### Definitions

> With this question type, users want to know what certain elements are. While this type of question should be addressed by the glossary, it's helpful to call out some basic definitions in the FAQ too (think of definitions for essential, recurring features in the product), and link out to the relevant part of the docs.
> 
> + **Question**
>     + Definition questions should start with this structure:
>         + What is/are...
> + **Answer**
>     + Think of a dictionary – short, concise, informational definitions help
>     + Link out to the glossary or other relevant docs if needed

Reviewer: ```suggestion
With this question type, users want to know what certain elements are. While this type of question should be addressed by the glossary, it is helpful to call out some basic definitions in the FAQ too (think of definitions for essential, recurring features in the product), and link out to the relevant part of the documentation.

+ **Question**
    + Definition questions should start with this structure:
        + What is/are...
+ **Answer**
    + Think of a dictionary – short, concise, informational definitions help
    + Link out to the glossary or other relevant documentation if needed
```

---
pcx_content_type: concept

> title: FAQ

Reviewer: the content is missing?
